src/neptune_scale/__init__.py

```diff
@@ -568,7 +568,7 @@ def log(
             run_id=self._run_id,
             step=step,
             timestamp=timestamp,
-            fields=configs,
+            configs=configs,
```

Comment:
hmm, this part is public, isn't it? is it alright to just change a parameter like that?